tests/posix/common/prj.conf

```diff
@@ -2,10 +2,15 @@ CONFIG_PTHREAD_IPC=y
 CONFIG_POSIX_API=y
 CONFIG_MAX_PTHREAD_COUNT=20
 CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
 CONFIG_SEM_VALUE_MAX=32767
 CONFIG_POSIX_MQUEUE=y
 CONFIG_HEAP_MEM_POOL_SIZE=4096
 CONFIG_MAX_THREAD_BYTES=4
 CONFIG_THREAD_NAME=y
 
 CONFIG_SMP=n
+
+CONFIG_NO_OPTIMIZATIONS=y
```

Comment:
why do we need those here? clearly not related to ztest API, no?

Reply:
Wow - that was really quick! Saw it as well and deleted already.